lisa/sut_orchestrator/azure/common.py

```diff
@@ -420,6 +421,21 @@ def check_or_create_resource_group(
                 )
 
 
+def get_location_from_resource_group(
```

Comment:
It's not used, is it useful? It's doesn't matter that where the resource group is.

Reply:
oh seems, I missed one commit

Reply:
deleted